.github/workflows/publish_latest_chart.yml

```diff
@@ -21,6 +21,10 @@ jobs:
       - name: Helm Add Repo
         run: helm repo add chartmuseum https://gitops:${{ secrets.HELM_PASSWORD }}@charts.fluvio.io
       - name: Push Sys Chart
-        run: helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)-latest" --force chartmuseum
+        run: |
+          helm push k8-util/helm/fluvio-sys --version="$(cat VERSION)" --force chartmuseum
```

Comment:
Do we need to publish release versions before releasing, is there a use case? Could this conflict with the release workflow?

Reply:
Oh I see, I didn't see the release workflow for helm. I'll get rid of the duplicate one with just VERSION